_authors/en/slaierno.md

```diff
@@ -6,7 +6,19 @@ lang: en
 lang-ref: slaierno
 ---
 
-This is my personal page.
+Born in 1990, raised as a [*terrone*](https://en.wikipedia.org/wiki/Terrone) and [bolognese](https://en.wikipedia.org/wiki/Bologna) by choice. I am one of those kids that always had a computer to play with.
 
-⚠ _Work in progress._ ⚠
+My house first computer was a [Commodore Amiga 500](https://en.wikipedia.org/wiki/Amiga_500). There is a picture of when I was a year and half old where I am on my father's lap while he was playing a videogame. Another picture, a year later, I was next to him, with the player 2 joystick controlling a jeep a giving support to my father's helicopter in [Silkworm](https://en.wikipedia.org/wiki/Silkworm_\(video_game\)).
```

Comment:
`The first computer in my house` sounds a bit better.

`controlling a jeep and giving support` on the last phrase.